extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -162,9 +162,22 @@ export class DoubleclickA4aEligibility {
    */
   isA4aEnabled(win, element, useRemoteHtml) {
     this.unconditionedExperimentSelection(win, element);
-    if ((useRemoteHtml && !element.getAttribute('rtc-config')) ||
-        'useSameDomainRenderingUntilDeprecated' in element.dataset ||
-        element.hasAttribute('useSameDomainRenderingUntilDeprecated')) {
+    const hasUSDRD =
+          'useSameDomainRenderingUntilDeprecated' in element.dataset ||
+          element.hasAttribute('useSameDomainRenderingUntilDeprecated');
+    if (hasUSDRD) {
+      user().warn(TAG, 'useSameDomainRenderingUntilDeprecated will no longer ' +
+                  'be supported starting on March 29, 2018. Please refer to ' +
+                  'https://github.com/ampproject/amphtml/issues/11834 ' +
+                  'for morch information');
+    }
+    if (useRemoteHtml) {
+      user().warn(TAG, 'The use of remote.html will no longer ' +
```

Comment:
Should we only send if useRemoteHtml & !RTC?

Reply:
I was thinking we should let them know either way. No harm to notify them if they have it

Reply:
ok